protocol-designer/src/components/StepEditForm/fields/DropTipField/index.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react'
+import { useSelector } from 'react-redux'
+import { DropdownField, DropdownOption, FormGroup } from '@opentrons/components'
+import { FLEX_TRASH_DEF_URI, OT_2_TRASH_DEF_URI } from '../../../../constants'
+import { i18n } from '../../../../localization'
+import {
+  getAdditionalEquipmentEntities,
+  getLabwareEntities,
+} from '../../../../step-forms/selectors'
+import { StepFormDropdown } from '../StepFormDropdownField'
+import styles from '../../StepEditForm.css'
+
+export function DropTipField(
+  props: Omit<React.ComponentProps<typeof StepFormDropdown>, 'options'>
+): JSX.Element {
+  const { value } = props
```

Comment:
using `something.value` in this component so probably using a different name would make code readable instead of `value`.

```suggestion
  const { value: dropDownItem } = props
```